server/tests/src/test/java/org/infinispan/server/security/AuthenticationAggregateRealmIT.java

```diff
@@ -0,0 +1,63 @@
+package org.infinispan.server.security;
+
+import static org.infinispan.client.rest.RestResponse.OK;
+import static org.infinispan.server.test.core.Common.assertStatus;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.infinispan.client.rest.RestClient;
+import org.infinispan.client.rest.configuration.RestClientConfigurationBuilder;
+import org.infinispan.commons.dataconversion.internal.Json;
+import org.infinispan.server.test.core.category.Security;
+import org.infinispan.server.test.junit4.InfinispanServerRule;
+import org.infinispan.server.test.junit4.InfinispanServerRuleBuilder;
+import org.infinispan.server.test.junit4.InfinispanServerTestMethodRule;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+/**
+ * @since 15.0
+ **/
+
+@Category(Security.class)
+public class AuthenticationAggregateRealmIT {
+   @ClassRule
+   public static InfinispanServerRule SERVERS =
+         InfinispanServerRuleBuilder.config("configuration/AuthenticationAggregateRealm.xml")
+               .numServers(1)
+               //.runMode(ServerRunMode.CONTAINER)
```

Comment:
Shouldn't we be making all tests run in CONTAINER mode by default now?

Reply:
My bad, it was a dev thing